modules/cudf/test/series/string-tests.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2020, NVIDIA CORPORATION.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {setDefaultAllocator} from '@nvidia/cuda';
+import {StringSeries} from '@rapidsai/cudf';
+import {CudaMemoryResource, DeviceBuffer} from '@rapidsai/rmm';
+import {Utf8Vector} from 'apache-arrow';
+
+const mr = new CudaMemoryResource();
+
+setDefaultAllocator((byteLength: number) => new DeviceBuffer(byteLength, mr));
+
+const data = [
+  'foo bar baz',   // start of string
+  ' foo bar baz',  // start of string after whitespace
+  'baz bar foo',   // end of string
+  'foo bar foo',   // start and end
+  'bar foo baz',   // middle
+  'baz quux',      // missing
+  'FoO',           // wrong case
+  'bar\n foo',     // multi-line
+  'bar\nfoo'       // multi-line
+];
+
+describe.each([['foo'], [/foo/], [/foo/ig]])('Series regex search (pattern=%p)', (pattern) => {
+  test('containsRe', () => {
+    const expected = [true, true, true, true, true, false, false, true, true];
+    const s        = StringSeries.new(Utf8Vector.from(data));
+    expect([...s.containsRe(pattern).toArrow()]).toEqual(expected);
```

Comment:
nit: Series implements `*[Symbol.iterator]()` method, so we can do this instead:
```suggestion
    expect([...s.containsRe(pattern)]).toEqual(expected);
```